server/routes/overview/index.ts

```diff
@@ -11,17 +11,19 @@ import PrisonerSupportNeedsRequestHandler from './prisonerSupportNeedsRequestHan
 export default (router: Router, services: Services) => {
   const prisonerSummaryRequestHandler = new PrisonerSummaryRequestHandler(services.prisonerSearchService)
   const prisonerSupportNeedsRequestHandler = new PrisonerSupportNeedsRequestHandler(services.curiousService)
-  const overViewController = new OverviewController()
+  const overViewController = new OverviewController(services.curiousService)
```

Comment:
I'm exploring a slightly different pattern here .... I'm wiring the curious service into the controller, so that the controller is responsible for getting (in this case) functional skills from Curious, rather than doing it in a request handler that stores it in the session

My thinking is that this data is a bit different to the main prisoner summary. The prisoner summary is used by most pages, so storing it in the session makes sense.
But data that is only shown on 1 tab (ie. functional skills on Education And Training; and possibly learning needs etc on Support Needs) does not need to go in the session (and in cases where we change prisoner etc, it might end up being wrong!)

Reply:
Makes sense, I'll do the same 👍